features/product/viewing_product_in_admin_panel/viewing_details_of_product_with_variants.feature

```diff
@@ -1,4 +1,4 @@
-@viewing_products
+@viewing_product_in_admin_panel
```

Comment:
Yes, of course we should update this suite name also in other scenarios, but I would be definitely for doing it in a separate PR. I will try to do it and if it will be too problematic, I will create a task for that 👌🏻

Reply:
Or I'll make a task right away to change this with the coverage of these scenarios in the API, because for now they are only covered in the UI